.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -392,7 +392,9 @@ jobs:
         uses: ./.github/actions/conn-disrupt-test-check
         with:
           job-name: ${{ env.job_name }}-${{ matrix.name }}-post-rotate
-          extra-connectivity-test-flags: ${{ steps.vars-conn.outputs.connectivity_test_defaults }}
+          # Opt-out from `no-ipsec-xfrm-error`, seeing XfrmOutPolBlock errors.
```

Comment:
Wait, why is this comment here? Shouldn't it be in the other file?

Reply:
Sure, that's a follow-up possibility. I just didn't want to go through a full cycle of CLI releases etc when we can already unbreak the unaffected workflows this way.

Reply:
:disappointed: